clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -91,6 +92,13 @@ export default async function handler(
   // Check if a geolocation was provided via headers or query param
   const geolocation = await lookupGeolocation(req);
 
+  const propertyId = verifyConditionsForPropertyId(
```

Comment:
nit: I'd expect a function called `verify***` to not also be a getter and just return a value.

Consider a better version of `safeLookupPropertyId` or `getPropertyIdButAlsoVerifyConditions` etc

Reply:
Fair point, `safeLookupPropertyId` sounds good to me